hail/python/hail/docs/hail_on_the_cloud.rst

```diff
@@ -5,7 +5,10 @@ Hail on the Cloud
 =================
 
 Public clouds are a natural place to run Hail, offering the ability to run
```

Comment:
I would prefer more direct phrasing, also "public" isn't really relevant here, since we're describing how to run on any cloud service provider (and depending on ingress rules, by default private).

Reply:
"Hail is designed to run on large computer clusters, so it's a great fit for Microsoft (Azure), Amazon (AWS), and Google (GCS) cloud platforms, which allow you to spin up clusters containing thousands of computers in minutes. And, we've designed Hail to be fault-tolerant, so using cheap, pre-emptible/spot instances has never been easier."


thoughts @cseed